app/policies/group_policy.rb

```diff
@@ -83,6 +83,15 @@ def change_leader?
     edit?
   end
 
+  def select_suite?
```

Comment:
I think it would change -- reps can assign as long as the group is in the next groups